unity-renderer/Assets/Rendering/ProceduralSkybox/ToolProceduralSkybox/Scripts/Editor/Procedural Skybox/SkyboxEditor.cs

```diff
@@ -186,7 +186,6 @@ private void Update()
 
         #endregion
 
-        [MenuItem("Window/Skybox Editor")]
         static void Init()
```

Comment:
This file is going to be removed after this commit is merged? There's a motive to have v2 and this one present on the codebase?

Reply:
Yes. As I'm also working on other branches where implementation is on the v1(Old UI) file. So removing this file will also delete v1 file on other branch while merging. I need this v1 file for the time being to copy other branches implementation to this new UI.